### DISCUSSION TITLE
Possible to preserve indent? Using Quarto in combination with Documenter.jl

### DISCUSSION THREAD

**Discussion by pat-alt** [quarto-dev/quarto-cli#2947] [Q&A]
I'm using Quarto in combination with [`Documenter.jl`](https://juliadocs.github.io/Documenter.jl/stable/) to build documentation for Julia packages (see for example [here](https://github.com/pat-alt/ConformalPrediction.jl)). The jury is still out on whether this is a good idea or an overkill, but so far I think it enables a good workflow. 

One nice feature of `Documenter.jl` is [Admonitions](https://juliadocs.github.io/Documenter.jl/stable/showcase/#Admonitions) (basically like Quarto's [Callout Blocks](https://quarto.org/docs/authoring/callouts.html)). The Markdown syntax for admonitions is as follows,

```
!!! note "An optional title"
    Here is something you should pay attention to.
```

where the four-space indent and wrapping is strictly necessary. In order to preserve the indent and wrapping I have tried the following:

```yml
format: 
  commonmark:
    wrap: preserve
```

While this does indeed preserve the wrapping, the indent is not preserved in the render `.md` document:

```
!!! note "An optional title"
Here is something you should pay attention to.
```

Is there a way to preserve the indent? Or perhaps there is an even better, standard way to just preserve the text as is?

Thanks in advance!

**Comment by cscheid** [comment]
Have you tried using this option in a call to plain Pandoc? That will help us track down if the issue is on quarto or pandoc.

FYI `quarto pandoc` will invoke the packaged version of pandoc, which is helpful in these testing scenarios.

**Reply by pat-alt** [reply]
Thanks @cscheid  

How do I call to plain Pandoc? I tried `quarto pandoc render /filepath/` but that doesn't work. Is there any guide that I haven't been able to find? See below for my pandoc version:

```
(base)  ~/code/ConformalPrediction.jl/ [develop*] quarto pandoc --version
pandoc 2.19.2
Compiled with pandoc-types 1.22.2.1, texmath 0.12.5.2, skylighting 0.13,
citeproc 0.8.0.1, ipynb 0.2, hslua 2.2.1
Scripting engine: Lua 5.4
User data directory: /Users/user/.local/share/pandoc
Copyright (C) 2006-2022 John MacFarlane. Web:  https://pandoc.org
This is free software; see the source for copying conditions. There is no
warranty, not even for merchantability or fitness for a particular purpose.
```

**Reply by dragonstyle** [reply]
Here is the user guide for Pandoc <https://pandoc.org/MANUAL.html> - the general approach would be to make an md file with the admonition and then:

```
quarto pandoc test.md --from markdown --to commonmark
```

That will output right to stdout, which should make it easy to see.

I did talk to our resident markdown expert and he noted that you can use lineblocks to preserve whitespace and that does appear to work. You'd write:

```markdown
| !!! Note "An optional title"
|     Here is something that you should pay attention to.   
```

which will preserve the whitespace and should result in the markdown you're looking for...

**Reply by pat-alt** [reply]
Thanks a lot, that's pretty much exactly what I was looking for! Another issue though is that quotation marks get converted from `"[title]"` to `“[title]”` (can barely spot the difference). Can I escape this somewhow?

Thanks again, both! 🙏🏽

**Reply by pat-alt** [reply]
Backslash seems to do the trick:

```
| !!! Note \"An optional title\"
|     Here is something that you should pay attention to.   
```
